plugins/woocommerce/includes/rest-api/Server.php

```diff
@@ -51,6 +51,12 @@ public function register_rest_routes() {
 					$legacy_proxy->get_instance_of( $controller_class );
 				$this->controllers[ $namespace ][ $controller_name ]->register_routes();
 			}
+			/**
+			 * Fires immediately after WooCommerce core routes have been registered for the given namespace.
+			 *
+			 * @since 10.3.0
+			 */
+			do_action( 'woocommerce_' . $namespace . '_rest_namespace_routes_registered' );
```

Comment:
Were these hooks introduced for a specific user need or reason? Just curious why they are needed.

Reply:
I'm going to remove this hook for now and revisit in a later PR.  If https://core.trac.wordpress.org/ticket/63946 gets adopted sooner than later, we can avoid adding this hook altogether.